app/models/methodology.rb

```diff
@@ -39,7 +43,7 @@ class Methodology
   # 'admin:paths:templates:methodologies' setting
   def self.pwd
     @pwd ||= begin
-      conf = Configuration.create_with(value: Rails.root.join('templates/methodologies/').to_s).
+      conf = Configuration.create_with(value: Rails.root.join('../../shared/templates/methodologies/').to_s).
```

Comment:
@brianp @dormi (!)

```suggestion
      conf = Configuration.create_with(value: Rails.root.join('templates/methodologies/').to_s).
```

Reply:
It's weird that it works without this change 🤔

Reply:
I tested it and with the change everything still (continues?) to work as expected.